specs/core/0_beacon-chain.md

```diff
@@ -864,7 +863,7 @@ For each `SpecialRecord` `obj` in `block.specials`, verify that its `kind` is on
 ```
 Perform the following checks:
 
-* Verify that `BLSVerify(pubkey=validators[data.validator_index].pubkey, msg=hash(LOGOUT_MESSAGE + bytes8(fork_version)), sig=data.signature, domain=get_domain(state, current_slot, DOMAIN_LOGOUT))`
+* Verify that `BLSVerify(pubkey=validators[data.validator_index].pubkey, msg="", sig=data.signature, domain=get_domain(state, current_slot, DOMAIN_LOGOUT))`
```

Comment:
In #141, BLSVerify was defined as `BLSVerify(pubkey: uint384, msg: bytes32, sig: [uint384], domain: uint64)`, so `msg=""` doesn't match the parameter. Maybe use `msg=bytes([0] * 32)`?
```suggestion
* Verify that `BLSVerify(pubkey=validators[data.validator_index].pubkey, msg=bytes([0] * 32), sig=data.signature, domain=get_domain(state, current_slot, DOMAIN_LOGOUT))`
```

Another solution is updating #141:
```python
def BLSVerify(pubkey: uint384, sig: [uint384], domain: uint64, msg: bytes32=None):
    if msg is None:
        msg = bytes([0] * 32
    ...
```